tests/components/tasmota/test_discovery.py

```diff
@@ -118,14 +100,14 @@ async def test_correct_config_discovery(
     # Verify device and registry entries are created
     device_entry = device_reg.async_get_device(set(), {("mac", mac)})
     assert device_entry is not None
-    entity_entry = entity_reg.async_get("switch.beer")
+    entity_entry = entity_reg.async_get("switch.test")
     assert entity_entry is not None
 
-    state = hass.states.get("switch.beer")
+    state = hass.states.get("switch.test")
     assert state is not None
-    assert state.name == "Beer"
+    assert state.name == "Test"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)